packaging/desktop/package_darwin.sh

```diff
@@ -138,6 +138,13 @@ get_deps() {(
   curl -J -L -Ss "$installer_url" | tar zx
 )}
 
+check_ci() {(
+  temp="$(mktemp -d)"
+  (cd "$temp" && npm i github-ci-status)
```

Comment:
the npm install is only in this packaging dir tree... its for appdmg and electron-package only used in building... This is not npm install for keybase/client/desktop :)